dart/example/example.dart

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
I initially had this in `sample/`, but moved it here.  I'd like it to be included in what's distributed on pub.dartlang.org, and this is a common convention for doing so (and will be more immediately helpful to people who install the package).  Do you want a copy of it also included in `sample/`?

Reply:
I certainly don't want copies. Dart packages can't pull from different directories?

I'd prefer it in `sample/` since there it can share the schema, and it is useful for people to see how FlatBuffers is used in different languages all in one place. Also, generally it is a structure we've used so far, and would be nice to not break for the sake of Dart.

Reply:
@aardappel: we are encouraging certain patters, and we have settled on the `example/` directory so far. You could script the publishing, and create a temp copy it from `sample/` just before calling `pub publish`, and remove it afterwards.

Reply:
I'd be fine with a publish script that copies things around, as long as they're in git in one location.

Reply:
My knowledge of Dart is minimal, so I'll leave it to you to judge. That said, we cannot have any duplicated files, and we already have 8 or so languages conforming to a particular pattern where tests and samples are in a particular locations and share particular files/schemas, so it be really good not to have to make an exception for Dart's sake.